modified.txt

```diff
@@ -0,0 +1,6 @@
+./tests/js/chartAuthoringTests.js
```

Comment:
Whoops - that was an accidental commit of a `find`-command file I generated to inventory the modified files for when I was updating the copyright date. Removed.